From f2701a7b8b962b614e014f6d37044e5a97af6ce1 Mon Sep 17 00:00:00 2001
From: Rahul T R <r-ravikumar@ti.com>
Date: Fri, 4 Aug 2023 21:50:37 +0530
Subject: [PATCH] arm64: dts: ti: k3-j721e-sk: Enable wkup i2c0

This is required ti configure PMIC, to fix the
issue of random reboots during log runs

Signed-off-by: Rahul T R <r-ravikumar@ti.com>

---
 arch/arm64/boot/dts/ti/k3-j721e-sk.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-j721e-sk.dts b/arch/arm64/boot/dts/ti/k3-j721e-sk.dts
index 039166b2b127..06852a42f958 100644
--- a/arch/arm64/boot/dts/ti/k3-j721e-sk.dts
+++ b/arch/arm64/boot/dts/ti/k3-j721e-sk.dts
@@ -650,6 +650,12 @@ &ospi1 {
 	status = "disabled";
 };
 
+&wkup_i2c0 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&wkup_i2c0_pins_default>;
+	status = "okay";
+};
+
 &main_i2c0 {
 	status = "okay";
 	pinctrl-names = "default";
